src/app/home/records/records-table/records-table.component.ts

```diff
@@ -30,6 +30,14 @@ export class RecordsTableComponent implements AfterViewInit {
     this.dataSource.data = this.data();
   }
 
+  refreshTableData(): void {
```

Comment:
but for that component you get data from parent component - record component and updating need to call from that component